src/tiles/item-image.ts

```diff
@@ -61,7 +63,7 @@ export class ItemImage extends LitElement {
 
   private get waveformTemplate() {
     return html`
-      <div class="box" id=${this.randomGradient}>
+      <div class="box">
```

Comment:
Since there's only one of these elements and you're specifically looking for it above, I would make this an `id` instead of a class. Classes are for when there are many elements of a similar style.

Also, `box` is a very generic name.. could you use something more specific?